client/src/modules/cohort/index.jsx

```diff
@@ -128,6 +130,16 @@ class Cohort extends PureComponent {
       : removeArchivedListsMetaData();
   };
 
+  handleAddNewMember = email => {
```

Comment:
can we handle this feature inside membersBox to keep all members actions in common/members components? e.g removing, changing roles are perform inside MemberBox... You will have access to the cohortId or listId there via route params...

Reply:
I don't think it is a good idea cause MemberBox is a generic component and it shouldn't now about business logic of cohort or lists. If I do your proposal I will have to fire addCohortMember inside MembersBox component.

I think that adding cohort member should be handled within the cohort module.